ansible/playbooks/tasks/cluster-bootstrap.yaml

```diff
@@ -230,18 +240,25 @@
     - use_sbd
     - is_primary
 
-- name: Configure AWS EC2 STONITH
-  ansible.builtin.command: >
-    crm configure primitive rsc_aws_stonith stonith:external/ec2
-    op start interval=0 timeout=180
-    op stop interval=0 timeout=180
-    op monitor interval=120 timeout=60
-    meta target-role=Started
-    params tag={{ aws_stonith_tag}} pcmk_delay_max=15
+- name: Configure AWS EC2 STONITH for sle 12sp5
+  ansible.builtin.command: "{{ aws_stonith_cmd }}"
+  when:
+    - is_primary
+    - cloud_platform_is_aws
+    - not (use_sbd | bool)
+    - ansible_distribution_major_version == '12'
+  register: stonith_config_result
+  failed_when: >
+    stonith_config_result.stderr_lines | select("match", "ERROR") | reject("match", "ERROR: warning") | list | length > 0
+
+
+- name: Configure AWS EC2 STONITH for sle 15
+  ansible.builtin.command: "{{ aws_stonith_cmd }}"
   when:
     - is_primary
     - cloud_platform_is_aws
     - not (use_sbd | bool)
+    - ansible_distribution_major_version != '12'
   register: stonith_config_result
   failed_when: "'ERROR' in stonith_config_result.stderr"
```

Comment:
not sure but maybe 
```suggestion
  failed_when: 'ERROR' in stonith_config_result.stderr
```

Reply:
I did that and it led to an error, and it was like this before, so I let it be